docs/how-to-monitor-minio-with-prometheus.md

```diff
@@ -71,3 +70,17 @@ Minio exposes the following metrics on `/minio/prometheus/metrics` endpoint. All
 - `minio_offline_disks` : Total number of offline disks for current Minio server instance
 - `minio_total_disks` : Total number of disks for current Minio server instance
 - `process_start_time_seconds` : Start time of the Minio server since unix epoch in seconds
+
+If you're running Minio gateway, disk/storage information is not exposed. Only the following metrics are available
+
+- `minio_http_requests_duration_seconds_bucket` : Cumulative counters for all the request types (HEAD/GET/PUT/POST/DELETE) in different time brackets
+- `minio_http_requests_duration_seconds_count` : The count of current number of observations i.e. total HTTP requests (HEAD/GET/PUT/POST/DELETE)
+- `minio_http_requests_duration_seconds_sum` : The current aggregate time spent servicing all HTTP requests (HEAD/GET/PUT/POST/DELETE) in seconds
+- `minio_network_received_bytes_total` : Total number of bytes received by current Minio server instance
+- `minio_network_sent_bytes_total` : Total number of bytes sent by current Minio server instance
+- `process_start_time_seconds` : Start time of the Minio server since unix epoch in seconds
```

Comment:
Shouldn't this be `uptime of the Minio server instance` ?

Reply:
Uptime is cumulative time minio server has been up since last reboot.  This sounds like start time.  We can drop 'the' here as well.